canonical-web-and-design/maas.io
# Further fix the contact us form

## Done
Remove the background submit thing from the general form

## QA
- Run the demo
- Go to /contract-us and fill in the form
- It should submit and return to the thank-you
- Go to the homepage
- Click contact-us in the header
- Check that works as expected
